nyancat/v0.16/README.md

```diff
@@ -0,0 +1,67 @@
+# Nyancat Testnet V0.16
+
+To support `DeFi` functionalities, we introduced the [HTLC](https://stage.irisnet.org/docs/features/htlc.html) and [Coin Swap](https://stage.irisnet.org/docs/features/coinswap.html) modules in v0.16, so that investors can transfer their tokens across chains and trade them decentralized.
+
+## Latest version: [v0.16.0-rc0](https://github.com/irisnet/irishub/releases/tag/v0.16.0-rc0)
+
+**Note**:
+
+- The point-to-`iris` conversion rate will be announced at the end of this incentivized testnet campaign.
+
+- For every task, only the top 40 completed validators can get rewards (the upgrade task is based on the voting list of the upgrade proposal, if a valid voter doesn't complete upgrade in time, the reward will be extended to the next completed validator)
+
+## v0.16.0-rc0
+
+As what we did on the testnets for v0.15, we will first test the new introduced functions of v0.16 on a new testnet `nyancat-5`, if everything works fine, then test the upgrade on `nyancat-6`.
```

Comment:
Just as we did..., we'll first test drive the new features of v0.16 ...; if everything works fine, we then go through the on-chain upgrade process ...